src/views/Dashboard/Customer/Home/RequestTest/ServiceDetailDialog.vue

```diff
@@ -103,10 +106,20 @@ export default {
     ...mapState({
       mnemonicData: (state) => state.substrate.mnemonicData,
       selectedService: (state) => state.testRequest.products
-    })  
+    }),
+
+    country () {
+      return this.countries.filter((c) => c.iso2 === this.selectedService.country)[0].name
```

Comment:
Just curious, it's not causing the error?

Reply:
no it's not